### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by hechmi-dammak-xenit**

### CODE REVIEW

File: xenit-aps/values.yaml [hunk 1]
```diff
@@ -4,174 +4,28 @@ general:
...
-    type: Recreate
-  resources:
-    requests:
-      memory: "512Mi"
-      cpu: "0.5"
-
-postgresql:
+      memory: "1Gi"
+      cpu: "1"
+postgresqlActivitiAdmin:
```
Comment (gert-glassee) [inline xenit-aps/values.yaml:28]: If this is a DB pod that requires persistent volumes the strategy.type should be Recreate.

File: xenit-aps/values.yaml [hunk 2]
```diff
@@ -184,139 +38,111 @@ postgresql:
...
 
-solr:
-  enabled: true
-  replicas: 2
-  podManagementPolicy: Parallel
+ingress:
+  protocol: 'http'
+  ingressAnnotations:
+    kubernetes.io/ingress.class: "nginx"
+    cert-manager.io/cluster-issuer: "letsencrypt-production"
```
Comment (gert-glassee) [inline xenit-aps/values.yaml:45]: Is this required for OUP?

File: xenit-aps/Chart.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-name: alfresco
+name: activiti
 description: A Helm chart for Alfresco managed by Xenit Solutions
```
Comment (kpiot123) [inline xenit-aps/Chart.yaml:3]: The description should mention that this helm chart is for APS. Now the description is the same as the one from helm-alfresco.

File: README.md
Comment (kpiot123) [inline README.md:1]: @hechmi-dammak-xenit 
Can you remove the following settings from the README?:
* persistentStorage.alfresco.enabled
* persistentStorage.alfresco.storageClassName
* persistentStorage.alfresco.storage
* persistentStorage.alfresco.efs.volumeHandle
* persistentStorage.alfresco.additionalClaims

I don't see them being used anywhere.

**Review by gert-glassee — APPROVED** [review]

**Review by kpiot123 — APPROVED** [review]

**Pull request merged**